src/questions/test-command.js

```diff
@@ -2,5 +2,12 @@ module.exports = projectInfos => ({
   type: 'input',
   message: '✅  Test command (use empty value to skip)',
   name: 'testCommand',
-  default: projectInfos.testCommand
+  default: answers => {
```

Comment:
If user does not have a `test` script in his package.json, it will still display it in the generated `package.json`(and we don't want that 😅 ) I'll take care of this part if you want